packages/manager/apps/web/client/app/hosting/hosting.routes.js

```diff
@@ -34,23 +34,25 @@ export default /* @ngInject */ ($stateProvider) => {
         $q,
         emailOptionIds,
         hostingEmailService,
+        isEmailDomainAvailable,
         OvhApiEmailDomain,
         serviceName,
       ) =>
-        $q
-          .all(
-            emailOptionIds.map((emailOptionId) =>
-              hostingEmailService
-                .getEmailOptionServiceInformation(serviceName, emailOptionId)
-                .then(
-                  ({ resource }) =>
-                    OvhApiEmailDomain.v6().serviceInfos({
-                      serviceName: resource.name,
-                    }).$promise,
-                ),
-            ),
-          )
-          .then((servicesInformation) => servicesInformation.flatten()),
+        (isEmailDomainAvailable
```

Comment:
Duplicate of #2960?

Reply:
Not entirely, as here we prevent making the call if we know it will throw an error